src/sideshift.js

```diff
@@ -0,0 +1,98 @@
+// @flow
+
+import type {StandardTx, SwapFuncParams} from './checkSwapService'
+
+const {checkSwapService} = require('./checkSwapService.js')
+const js = require('jsonfile')
+const fetch = require('node-fetch')
+
+const confFileName = './config.json'
+const config = js.readFileSync(confFileName)
+
+const SIDESHIFT_CACHE = './cache/XAIRAW.json'
+const PAGE_LIMIT = 500
+
+type SideShiftTransaction = {
+  quoteId: string,
```

Comment:
use `orderId`

Reply:
currently the endpoint return an `id` and a `quoteId`. `id` will be changed to `orderId` or should I just use `id` here?

Reply:
My bad, use `id` from the response. We're moving away from both `orderId` and `quoteId`. Especially `quoteId`, because we now have separated quotes/orders. (transactions represent orders, not quotes)

Reply:
Currently, `quoteId` is only there for backwards compatibility. API consumers should move away from using `quoteId` to refer to `Orders`. They're no longer interchangeable.

In the future, `quoteId` will point to the actual `Quote` that was accepted to create the `Order`.